src/Forms/ValidatorList.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+namespace SilverStripe\Forms;
+
+use InvalidArgumentException;
+use SilverStripe\ORM\ValidationResult;
+
+/**
+ * Class ValidatorList
```

Comment:
Docblock please – we're introducing a public API that will be supported for years to come.